addons/website_forum/views/forum_forum_templates_layout.xml

```diff
@@ -90,7 +90,7 @@
                     <i class="fa fa-navicon"/>
                 </button>
             </div>
-            <div t-attf-class="d-flex justify-content-lg-end gap-2 flex-grow-1 w-100 w-lg-auto #{'mw-xl-75' if search else 'mw-xl-50'}">
+            <div t-attf-class="d-flex justify-content-lg-end gap-2 flex-grow-1 flex-wrap flex-lg-nowrap w-100 w-lg-auto #{'mw-xl-75' if search else 'mw-xl-50'}">
```

Comment:
I'd go with `flex-md-nowrap`, otherwise the "New Post" Button is too big when screen is higher than md.